doc/classes/@GlobalScope.xml

```diff
@@ -953,181 +953,181 @@
 			Extra mouse button 2 mask.
 		</constant>
 		<constant name="JOY_INVALID_BUTTON" value="-1" enum="JoyButtonList">
-			An invalid game controller button.
+			Invalid button: An invalid game controller button.
 		</constant>
 		<constant name="JOY_BUTTON_A" value="0" enum="JoyButtonList">
-			Game controller SDL button A.
+			Bottom Action: Game controller SDL button A.
```

Comment:
Should we rename those "canonical controller" entries in `JoyButtonList` to match this standardize naming, and replace those which duplicate `JOY_XBOX_A`, etc.?